front/app/components/Avatar/index.tsx

```diff
@@ -264,13 +264,15 @@ const AvatarInner = ({
       )}
 
       {verified && addVerificationBadge && (
-        <FeatureFlag name="verification">
-          <BadgeIcon
-            name="check-circle"
-            size={badgeSize}
-            fill={colors.success}
-          />
-        </FeatureFlag>
+        <Suspense fallback={null}>
```

Comment:
@luucvanderzee I wasn't sure about the fallbacks, so I have set them all to `null`. I am guessing this is not right, however (in any of the files changed here)